packages/swirl-components/src/components/swirl-image-grid/swirl-image-grid.tsx

```diff
@@ -23,6 +23,10 @@ export class SwirlImageGrid {
       this.el.children
     ) as HTMLSwirlImageGridItemElement[];
 
+    this.items.forEach((item) => {
+      item.overlay = undefined;
```

Comment:
Issue: This would break other usages, right?

Reply:
What usages do you mean? 🤔 
It still sets the overlay in the following lines, but now it resets them before doing that

Reply:
Just saw your description, and the lines below. Nevermind.